docker-compose.yml

```diff
@@ -47,15 +47,100 @@ services:
       n1l1tos1:
         ipv4_address: 10.168.1.5
       n2htoleaf1:
-      n3htoleafbn1:
-      n5h1tol1r:
       n6h1tol1y:
     command: |
       sh -x -c 'touch /etc/frr/vtysh.conf && \
             sed -i "s/bgpd=no/bgpd=yes/g" /etc/frr/daemons && \
             sed -i "s/127.0.0.1/0.0.0.0/g" /etc/frr/daemons && \
             ip link add name lo0 type dummy && \
             ifconfig lo0 10.0.0.2 netmask 255.255.255.255 up && \
+
+            ip link add br-tenant type bridge vlan_default_pvid 0 vlan_filtering 1 vlan_protocol 802.1Q && \
+            ip link set br-tenant up && \
+
+            ip link add blue type vrf table 1000 && \
+            ip link set blue up && \
+            ip link add name lo1 type dummy && \
+            ip link set lo1 master blue up && \
```

Comment:
this code adds dummy iface
```
ip link add name lo1 type dummy && \
ip link set lo1 master blue up && \
ip address add 10.0.1.2/32 dev lo1 && \
```
is different from @mardim91 setting IP on the VRF itself...
```
ip address add 10.0.1.2/32 dev blue 
```